assets/js/base/components/button/index.tsx

```diff
@@ -44,6 +44,10 @@ export interface ButtonProps extends WPButton.ButtonProps {
 	 * Button variant
 	 */
 	variant?: 'text' | 'contained' | 'outlined';
+	/**
+	 * Button href
+	 */
+	href?: string;
```

Comment:
If this is an optional prop, then we should define it like so:

```suggestion
	href?: string | undefined;
```

This is because we have https://www.typescriptlang.org/tsconfig/exactOptionalPropertyTypes.html enabled.

Reply:
Right, I recall that Luigi made that change a while ago. I just adjusted this part accordingly.